tests/monotouch-test/AudioToolbox/AudioComponentTest.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright 2011 Xamarin Inc. All rights reserved
+
+#if !__WATCHOS__
+using System;
+using System.Drawing;
+using System.IO;
+using System.Collections.Generic;
+#if XAMCORE_2_0
+using Foundation;
+using AudioToolbox;
+using AudioUnit;
+using CoreFoundation;
+#else
+using MonoTouch.Foundation;
+using MonoTouch.MediaPlayer;
+using MonoTouch.AudioToolbox;
+using MonoTouch.CoreFoundation;
+#endif
+using NUnit.Framework;
+
+namespace MonoTouchFixtures.AudioToolbox {
+	
+	[TestFixture]
+	[Preserve (AllMembers = true)]
+	public class AudioComponentTest {
+
+		[Test]
+		public void GetSetComponentList ()
+		{
+			if (IntPtr.Size == 4)
+				Assert.Ignore ("API only available in 64 bits.");
```

Comment:
The API isn't 32-bit only, it's iOS 11 (and equivalent versions for other platforms) only.

This means you should use `TestRuntime.AssertXodeVersion (9, 0)` instead.